### DISCUSSION TITLE
flit and flash?

### DISCUSSION THREAD

**Discussion by rwijtvliet** [LazyVim/LazyVim#2931] [Q&A]
In the [mappings](http://www.lazyvim.org/keymaps), I saw lazyvim uses both flit and flash. Is that intentional? They seem to fulfill the same functionality.

**Comment by dpetka2001** [comment]
It uses only `flash.nvim`. `flit` gets enabled when you enable the `leap.nvim` LazyExtra and in that case `flash` gets disabled, so there's no conflicting keymaps.

**Comment by rwijtvliet** [comment]
Ah - understood. Many thanks for the help!
